README.md

```diff
@@ -1,2 +1,47 @@
-# flySorter-back-end-v2     
+![logo](./assets/flysorter-logo.png)
+# Welcome
+The following documentation is specifically for the ***BACK END*** repository. If you are looking for the ***FRONT END*** respository click [HERE](https://github.com/fly-sorter/flySorter-front-end-v2).
 
+# Description
+FlySorter is an inventory management application for the Seattle-based company: [FlySorter](https://flysorter.com/). FlySorter builds machines to sort fruit flies so they can be used for scientific research. This application keeps track of those machine assemblies, sub-assemblies, and the parts that belong to those sub assemblies. These components are displayed on the page and allows the user to click on a sub-assembly to reveal the parts that makeup that sub-assembly. The table also allows the user to filter parts and search for a part in order to make finding particular parts faster and easier for the user.
+
+# Why It Matters
+_"In 1908, looking to test Mendel’s theories on heredity, Thomas Hunt Morgan began studying Drosophila melanogaster (also known as the fruit fly or vinegar fly) in his “Fly Room” at Columbia University. Dr. Morgan and his students studied flies, looking for mutations, and found that some individuals had white eyes instead of the usual red eyes. Experimentation on the prevalence and propagation of the mutation led to the first proof that the chromosomal theory of inheritance was correct, and established drosophila as the first model research organism. The use of drosophila has been extremely fruitful in the past 100 years. Since Dr. Morgan’s experiments, drosophila research has been featured in over 100,000 scientific papers. This includes being involved in Nobel prize-winning research five times (in 1933, 1946, 1995, 2004, and 2011)."_ — [The Research Value of Drosophila Melangoaster](http://powersscientific.com/the-research-value-of-drosophila-melanogaster/)
+
+# Languages, Technologies, Frameworks, & Libraries Used
+The following technologies were used to develop the backend:
+
+1. AWS [Lambda](https://aws.amazon.com/lambda/) running NodeJS 6.10
+2. AWS [Aurora](https://aws.amazon.com/rds/aurora/) running Serverless MySQL
+3. AWS [Cognito](https://aws.amazon.com/cognito/) for authentication
```

Comment:
FYI:  Cognito gets used for both authentication AND authorizartion. 
May be appropriate to also add in IAM as it handles the security between services.